### DISCUSSION TITLE
It would be great if we can retrieve reviews content from API

### DISCUSSION THREAD

**Discussion by robinchen123** [amzn/selling-partner-api-models#615] [Ideas] [answered]
As many vendors or sellers are concerned about their product performance. Reviews is a quite important guide to know what's the weakness. It would be great if we can retrieve reviews content from API

**Comment by MarekMalczewskiWavemaker** [comment]
That's true. The content of review is one thing that is going to be extremely useful. Another one is average product rating.

**Reply by npatten** [reply]
Agreed - my team would really like to see average product ratings via the API!

**Comment by krithic15** [comment, marked as answer]
Hello @robinchen123 and @MarekMalczewskiWavemaker 
Thank you for your suggestion. We have submitted your feature request to the appropriate team.
Thanks,
Krithi, Solutions Architect
Selling Partner Developer Services

**Comment by krithic15** [comment]
> As many vendors or sellers are concerned about their product performance. Reviews is a quite important guide to know what's the weakness. It would be great if we can retrieve reviews content from API

@robinchen123,
Can you please let us know how you currently access this data?

**Reply by MarekMalczewskiWavemaker** [reply]
I am not sure if I understand your question correctly. As far as me is concerned I use Selling Partner Catalog API (https://sellingpartnerapi-na.amazon.com/catalog/2020-12-01/items/<ASIN>) to obtain product details.

**Reply by krithic15** [reply]
@MarekMalczewskiWavemaker May I know how you currently access the product reviews?

**Reply by MarekMalczewskiWavemaker** [reply]
I don't
